### ISSUE TITLE
#ELI5: Explain it like I'm 5 Lightning Talks

### ISSUE THREAD

**Issue by neozenith** [newwwie/newcastlejs-meetup#7]
## Title

ELI5: Explain it like I'm 5 Lightning Talks

## Description

Ever been confused by terms and technologies in the rapidly changing Javascript and Web Development Ecosystem? Now is the time to rally some lightning talks and _Explain It Like I'm a 5 Year Old_.

Use the comments section for topics and 👍  vote for topics. If you'd like to claim a topic for a lightning talk put your hand up and we'll update this top level description.

## Presenter Bio

So far:

| Talk | Speaker |
| --- | --- |
| JWTs | @KleeUT |
| Headless CMS | @david-boyd  |
| CDNs | @caevyn  |
| Typescript | @sheakelly  |

## Promo Photos

N/A

----

## Duration

Lightning talks, 5 min ea

## Target Audience
Beginners

## Hardware

We will have a laptop available to google reference materials if you don't want to bring your own hardware.

## Date
Aim for 8th May

## Misc

See comments for topics to upvote or claim or suggest a new topic.
 
----

**Comment by neozenith** [comment]
# Typescript

**Comment by neozenith** [comment]
# async / await

**Comment by neozenith** [comment]
# GraphQL

**Comment by neozenith** [comment]
# Content Distribution Networks

**Comment by neozenith** [comment]
# Headless CMS

**Comment by neozenith** [comment]
# Progressive Web Apps

**Comment by neozenith** [comment]
# JWTs

**Comment by david-boyd** [comment]
And I will put my hand up for Headless CMS

**Comment by neozenith** [comment]
Awesome thanks! @KleeUT, @david-boyd, @caevyn, @sheakelly 

Aim for Wednesday 8th May, 5 - 10 minutes lightning talk?

I'll keep this issue open in case we get enough responses for a second session perhaps incorporating GraphQL or NativeScript

**Comment by neozenith** [comment]
# NativeScript / ReactNative

**Comment by neozenith** [comment]
@KleeUT @david-boyd @caevyn @sheakelly 

Are we still good for the 8th May? Would you prefer if we shift organising this to Slack or keep it on here?

We are going to need a few things though:
1. Confirmation you're still good for that date
1. We won't need a talk description because that is going to be self explanatory.
1. A summary sentence about yourself
1. A promo photo we can use for the event.

Thanks again for volunteering!!!

**Comment by KleeUT** [comment]
Still good. Let's move to slack

**Comment by sheakelly** [comment]
Yes I am still keen. Happy to chat via slack. is there a specific channel?

**Comment by jnowland** [comment]
Thanks for everyone who spoke!
